app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -56,3 +56,28 @@ ul.tabular {
   margin-bottom: 18px;
   margin-top: -8px;
 }
+
+.panel-workflow {
+  background: #FFF;
+  border-color: #e0e0e0;
+  left: 0;
+  margin: 0 auto;
+  max-width: 1140px;
```

Comment:
Why are we setting all these directives and not just using `class="panel-default"` or `@extend .panel-default`?

Reply:
I don't want to extend `.panel-default` as it would throw of the use of it for the show page itself. And I would want to avoid overriding any of the .panel-* configs to match this mockup in case they are used later with unexpected results.